manifest.konnector

```diff
@@ -1,18 +1,15 @@
 {
   "version": "1.0.0",
-  "name": "konnector name",
-  "type": "node",
-  "slug": "konnectorslug",
-  "description": "description",
-  "source": "git://github.com/konnectors/cozy-konnector-thename.git",
+  "name": "Boulanger",
+  "type": "io.cozy.bills",
+  "slug": "boulanger",
+  "description": "Konnector for the retailer Boulanger",
+  "source": "git://github.com/konnectors/boulanger.git",
   "locales": {
     "fr": {
       "permissions": {
-        "books": {
-          "description": "Utilisé pour sauvegarder les références de vos livres"
-        },
-        "files": {
```

Comment:
You also need to add : 

- io.cozy.files permission to allow the connector to save the pdf bills
- io.cozy.bank.operations to be able to link bills with operations

Like https://github.com/konnectors/darty/blob/b5f416281184fc6bd89a91f8ce7c68ef23192818/manifest.konnector#L27-L43

Reply:
Done